hackathon_site/dashboard/frontend/src/slices/event/teamSlice.ts

```diff
@@ -96,3 +95,16 @@ export const teamSelector = createSelector(
     [teamSliceSelector],
     (teamSlice) => teamSlice.team
 );
+
+export const teamCodeSelector = createSelector(
```

Comment:
you need to add tests to the teamslice.test.ts for these selectors. should be a small change